changelog/9.0.2.asciidoc

```diff
@@ -0,0 +1,43 @@
+// begin 9.0.2 relnotes
+
+[[release-notes-9.0.2]]
+==  9.0.2
+
+Review important information about the  9.0.2 release.
+
+
+
+
+
+
+
+
+
+[discrete]
+[[new-features-9.0.2]]
+=== New features
+
+The 9.0.2 release adds the following new and notable features.
+
+
+fleet-server::
+
+* Migrate-Action. {fleet-server-pull}https://github.com/elastic/fleet-server/pull/4786[#https://github.com/elastic/fleet-server/pull/4786] 
++
+Added new MIGRATE action type for moving agent to different cluster.
```

Comment:
This isn't usable yet and probably shouldn't have been backported because it's a new feature.

I could reword this but it's easier to just delete it because nobody can use it yet.